[2025-01-20T10:00:14.209+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-20T10:00:14.239+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cryptobot_workflow.fetch_data manual__2025-01-20T10:00:01.735068+00:00 [queued]>
[2025-01-20T10:00:14.260+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cryptobot_workflow.fetch_data manual__2025-01-20T10:00:01.735068+00:00 [queued]>
[2025-01-20T10:00:14.261+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-20T10:00:16.532+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): fetch_data> on 2025-01-20 10:00:01.735068+00:00
[2025-01-20T10:00:16.656+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=262) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-20T10:00:16.657+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'cryptobot_workflow', 'fetch_data', 'manual__2025-01-20T10:00:01.735068+00:00', '--job-id', '67', '--raw', '--subdir', 'DAGS_FOLDER/cryptobot_dag.py', '--cfg-path', '/tmp/tmpfb5fusiq']
[2025-01-20T10:00:16.666+0000] {standard_task_runner.py:105} INFO - Job 67: Subtask fetch_data
[2025-01-20T10:00:16.669+0000] {standard_task_runner.py:72} INFO - Started process 263 to run task
[2025-01-20T10:00:20.801+0000] {task_command.py:467} INFO - Running <TaskInstance: cryptobot_workflow.fetch_data manual__2025-01-20T10:00:01.735068+00:00 [running]> on host 03c752f9c1ed
[2025-01-20T10:00:21.073+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='cryptobot_workflow' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-01-20T10:00:01.735068+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-20T10:00:01.735068+00:00'
[2025-01-20T10:00:21.084+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-20T10:00:21.086+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-20T10:00:21.087+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-01-20 10:00:14.240481+00:00
[2025-01-20T10:00:21.088+0000] {logging_mixin.py:190} INFO - Dag name:cryptobot_workflow and current dag run status:running
[2025-01-20T10:00:21.089+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-20T10:00:21.105+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-01-20T10:00:21.109+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/src/extract.py']
[2025-01-20T10:00:21.141+0000] {subprocess.py:99} INFO - Output:
[2025-01-20T10:00:42.485+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.493+0000] {subprocess.py:106} INFO - Error processing BTCUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.494+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.495+0000] {subprocess.py:106} INFO - Error processing ETHUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.496+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.497+0000] {subprocess.py:106} INFO - Error processing SOLUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.498+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.500+0000] {subprocess.py:106} INFO - Error processing BNBUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.500+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.501+0000] {subprocess.py:106} INFO - Error processing XRPUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.501+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.502+0000] {subprocess.py:106} INFO - Error processing DOGEUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.502+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.502+0000] {subprocess.py:106} INFO - Error processing ADAUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.503+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.504+0000] {subprocess.py:106} INFO - Error processing TRXUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.504+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.504+0000] {subprocess.py:106} INFO - Error processing AVAXUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:42.504+0000] {subprocess.py:106} INFO - No more data returned, breaking the loop.
[2025-01-20T10:00:42.505+0000] {subprocess.py:106} INFO - Error processing TONUSDT: cannot access local variable 'documents' where it is not associated with a value
[2025-01-20T10:00:43.003+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-01-20T10:00:43.064+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-20T10:00:43.065+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cryptobot_workflow, task_id=fetch_data, run_id=manual__2025-01-20T10:00:01.735068+00:00, execution_date=20250120T100001, start_date=20250120T100014, end_date=20250120T100043
[2025-01-20T10:00:43.083+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-20T10:00:43.083+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-20T10:00:43.083+0000] {logging_mixin.py:190} INFO - Dag name:cryptobot_workflow queued_at:2025-01-20 10:00:01.817244+00:00
[2025-01-20T10:00:43.083+0000] {logging_mixin.py:190} INFO - Task hostname:03c752f9c1ed operator:BashOperator
[2025-01-20T10:00:43.133+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-20T10:00:43.186+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-20T10:00:43.187+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
